.github/workflows/issue-remediate.yml

```
@@ -0,0 +1,188 @@
+    steps:
+      - name: Checkout repository code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+
+      # Fail fast instead of letting the agent run for minutes and then 403 at
+      # PR creation. This job has read-only permissions, so only a PAT can open
+      # the fix PR (and PAT-created PRs are what trigger the normal CI).
+      - name: Require AUTOPILOT_TOKEN
+        if: ${{ secrets.AUTOPILOT_TOKEN == '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)